api/src/graphql/playground.ts

```diff
@@ -0,0 +1,63 @@
+export function renderGraphqlPlayground(
+  url: string,
+  headers: Record<string, string> = {}
+): string {
+  return `<!doctype html>
```

Comment:
For now, I've just added an html snippet that imports React and Graphqil from [unpkg](https://unpkg.com).
In the future if we have the need to customize the playground we can start bundling it.